style/patch.ts

```diff
@@ -42,3 +42,46 @@ for (const mode of ["light", "dark"]) {
 }`,
   );
 }
+
+const command = new Deno.Command("npx", {
+  args: ["parce", "build", "main.scss", "--no-source-maps"],
```

Comment:
I assume this is supposed to be:
```suggestion
  args: ["parcel", "build", "main.scss", "--no-source-maps"],
```

Reply:
Not sure how this worked on my machine? Nice catch.